ansible/modules/hashivault/hashivault_read.py

```diff
@@ -72,6 +72,7 @@ def main():
     argspec['secret'] = dict(required=True, type='str')
     argspec['key'] = dict(required=False, type='str')
     argspec['register'] = dict(required=False, type='str')
+    argspec['okifmissing'] = dict(required=False, default=False, type='str')
```

Comment:
what is the type for a boolean, bool?  I also feel there must be some more ansible way of doing this and I need to research that.  One thing that comes to mind is a default value if it doesn't exist, but not sure that is the right approach.